articles/api-auth/grant/hybrid.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Describes how to call APIs from applications using the Hybrid Flow
+topics:
+  - authorization-code
+  - api-authorization
+  - implicit
+contentType: concept
+useCase:
+  - secure-api
+  - call-api
+---
+# Call APIs Using the Hybrid Flow
+
+The Hybrid Flow is an OpenID Connect (OIDC) flow that draws from the following:
+
+1. [Authorization Code Grant](/api-auth/grant/authorization-code)
+2. [Implicit Grant](/articles/api-auth/grant/implicit)
+
+The Hybrid Flow enables use cases where your application can immediately use an ID token to access information about the user while obtaining an authorization code that can be exchanged for an Access Token (therefore gaining access to protected resources for an extended period of time).
+
+## Background
+
+With the [Authorization Code Grant](/api-auth/grant/authorization-code), Auth0 sends you an authorization code (authorization not required), which your app then sends in to retrieve tokens (authorization required). Your application authorizes itself with a Client ID and Client Secret stored securely on your server.
```

Comment:
items in parentheses are confusing, i'd recommend to remove the parentheses and their contents altogether or instead of "authorization" use "client authentication" which is what I believe is the intent.

Reply:
"Your application authorizes" > "Your application authenticates"